// @care/utils/dist/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var dateFormat_1 = require("./dateFormat");
exports.dateFormat = dateFormat_1.default;
var getQueryByName_1 = require("./getQueryByName");
exports.getQueryByName = getQueryByName_1.default;
var compareStrings_1 = require("./compareStrings");
exports.compareStrings = compareStrings_1.default;
var eventListener_1 = require("./eventListener");
exports.EventListener = eventListener_1.default;
var makePromiseCancelable_1 = require("./makePromiseCancelable");
exports.makePromiseCancelable = makePromiseCancelable_1.default;
var createBaseUrl_1 = require("./createBaseUrl");
exports.createBaseUrl = createBaseUrl_1.default;
var getBase64Image_1 = require("./getBase64Image");
exports.getBase64Image = getBase64Image_1.default;
var getImage_1 = require("./getImage");
exports.getImage = getImage_1.default;
var getPx_1 = require("./getPx");
exports.getPx = getPx_1.default;
var px2rem_1 = require("./px2rem");
exports.px2rem = px2rem_1.default;
var handleStylePx2rem_1 = require("./handleStylePx2rem");
exports.handleStylePx2rem = handleStylePx2rem_1.default;
var timeoutPromise_1 = require("./timeoutPromise");
exports.timeoutPromise = timeoutPromise_1.default;
var retryPromise_1 = require("./retryPromise");
exports.retryPromise = retryPromise_1.default;
var isHttp_1 = require("./isHttp");
exports.isHttp = isHttp_1.default;
var isPhoneNum_1 = require("./isPhoneNum");
exports.isPhoneNum = isPhoneNum_1.default;
var copyTextToClipBoard_1 = require("./copyTextToClipBoard");
exports.copyTextToClipBoard = copyTextToClipBoard_1.default;
var makePageResumeToBottomWhenInputBlur_1 = require("./makePageResumeToBottomWhenInputBlur");
exports.makePageResumeToBottomWhenInputBlur = makePageResumeToBottomWhenInputBlur_1.default;
var unFormatPhoneValue_1 = require("./unFormatPhoneValue");
exports.unFormatPhoneValue = unFormatPhoneValue_1.default;
var parseRestUrl_1 = require("./parseRestUrl");
exports.parseRestUrl = parseRestUrl_1.default;
var deviceInfo_1 = require("./deviceInfo");
exports.getDeviceId = deviceInfo_1.getDeviceId;
exports.generateDeviceId = deviceInfo_1.generateDeviceId;
var loadingEnhance_1 = require("./loadingEnhance");
exports.loadingDecorator = loadingEnhance_1.loadingDecorator;
exports.loadingEnhance = loadingEnhance_1.loadingEnhance;
var file_util_1 = require("./file-util");
exports.dataURL2Blob = file_util_1.dataURL2Blob;
var download_file_1 = require("./download-file");
exports.downloadFile = download_file_1.default;
exports.url2Blob = download_file_1.url2Blob;
exports.saveBlobAsFile = download_file_1.saveBlobAsFile;
var trimStr_1 = require("./trimStr");
exports.trimStr = trimStr_1.default;
